.github/workflows/laravel.yml

```
@@ -106,11 +108,23 @@ jobs:
+    - name: Copy environment file
+      run: cp .env.example .env
+
+    - name: Generate application key
+      run: php artisan key:generate
+
+    - name: Generate JWT secret
+      run: php artisan jwt:secret
+
     - name: Run Laravel Pint (Code Style)
       run: ./vendor/bin/pint --test
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)